examples/routers/table.vue

```diff
@@ -62,6 +62,9 @@
         methods: {
             handleClearCurrentRow () {
                 this.$refs.currentRowTable.clearCurrentRow();
+            },
+            onDragDrop(a,b){
+                console.log(a,b)
```

Comment:
?

Reply:
in examples catalog, can use `console` to test

Reply:
console.log(typeof a) => "string"
console.log(typeof b)  => "number"
a and b belong to different types

Reply:
> in examples catalog, can use `console` to test

console.log(typeof a) => "string"
console.log(typeof b) => "number"
a and b belong to different types